internal/tools/pipeline/dbclient/op_pipeline.go

```diff
@@ -367,10 +366,11 @@ func (client *Client) PageListPipelines(req *pipelinepb.PipelinePagingRequest, o
 			forceIndexes = append(forceIndexes, "`idx_id_source_cluster_status`")
 		}
 	}
+
 	// idx_id_source_cluster_status_timebegin_timeend
-	// 使用 alias 注入实现 xorm 插入 FORCE INDEX
-	if len(forceIndexes) > 0 {
-		baseSQL.Alias(fmt.Sprintf("`%s` USE INDEX (%s)", (&spec.PipelineBase{}).TableName(), strings.Join(forceIndexes, ",")))
+	for _, index := range forceIndexes {
+		// this method only support use in mysql
+		baseSQL.IndexHint("USE", "", index)
```

Comment:
Why do you need an empty string in the middle?

Reply:
This empty param is `forType`, used in the statement `USE INDEX FOR 'forType'(index_list)`
Our original usage was simply `USE INDEX(index_list)`, so set it as empty.
The usage can refer to the MySQL's documentation.
<img width="940" alt="image" src="https://github.com/erda-project/erda/assets/86157286/33fffcb9-98e5-4048-adf6-a9911fca83fa">